src/test/integration/IntegrationChecks.t.sol

```diff
@@ -1,23 +1,65 @@
 // SPDX-License-Identifier: BUSL-1.1
 pragma solidity ^0.8.27;
 
-import "src/test/integration/IntegrationBase.t.sol";
+import "src/test/integration/IntegrationUtils.t.sol";
 import "src/test/integration/users/User.t.sol";
 import "src/test/integration/users/User_M1.t.sol";
 import "src/test/integration/users/User_M2.t.sol";
 
 /// @notice Contract that provides utility functions to reuse common test blocks & checks
-contract IntegrationCheckUtils is IntegrationBase {
+contract IntegrationChecks is IntegrationUtils {
     using ArrayLib for *;
-    using SlashingLib for *;
     using StdStyle for *;
 
+    AVS avs;
+    SlashingParams slashParams;
+    SlashingParams slashingParams;
+
+    User operator;
+    OperatorSet operatorSet;
+    AllocateParams allocateParams;
+
+    User staker;
+    IERC20[] tokens;
+    IStrategy[] strategies;
+    uint[] initTokenBalances;
+    uint[] initDepositShares;
+    uint[] withdrawableShares;
+    // Withdrawal[] withdrawals; // cannot copy from memory to storage easily
+    bytes32[] withdrawalRoots;
+    uint[] numTokensRemaining;
+
+    uint64 beaconBalanceGwei;
+    uint40[] validators;
+    uint40[] slashedValidators;
+    uint64 slashedGwei;
+    uint64 slashedAmountGwei;
```

Comment:
What are all these state variables?

They're shadowed throughout the contract -- can we get rid of them? Tests that need persistent state across setups can just declare it themselves.

(I know we have a bunch of tests that declare similar storage, but people are gonna declare storage variables no matter what - I think it's good to have that at the surface level)